locksmith/src/controllers/v2/claimController.ts

```diff
@@ -12,28 +17,38 @@ export const LOCKS_WITH_DISABLED_CLAIMS = [
   '0xafcfff71f3e717fcdb0b6a1bf20026304fd41bee',
 ]
 
-export const claim: RequestHandler = async (request, response) => {
+/**
+ * Claim API on free locks on chains with low transaction fees.
+ * This also supports walletless airdrops.
+ * @param request
+ * @param response
+ * @returns
+ */
+export const claim: RequestHandler = async (request, response: Response) => {
   const { data } = await ClaimBody.parseAsync(request.body)
   const network = Number(request.params.network)
   const lockAddress = normalizer.ethereumAddress(request.params.lockAddress)
-  const owner = normalizer.ethereumAddress(request.user!.walletAddress)
-  // First check that the lock is indeed free and that the gas costs is low enough!
-  const pricer = new KeyPricer()
-  const pricing = await pricer.generate(lockAddress, network)
-  const fulfillmentDispatcher = new Dispatcher()
+  let owner = normalizer.ethereumAddress(request.user!.walletAddress)
+  const email = request.params.email?.toLowerCase()
```

Comment:
shouldn't this be query or body instead of param?

Reply:
yes! you are correct :)